src/sys/socket/sockopt.rs

```diff
@@ -233,6 +233,17 @@ cfg_if! {
         sockopt_impl!(SetOnly, Ipv6DropMembership, libc::IPPROTO_IPV6, libc::IPV6_LEAVE_GROUP, super::Ipv6MembershipRequest);
     }
 }
+cfg_if! {
```

Comment:
`cfg_if` is helpful when you expect to have `else` or `else if` blocks.  But in cases like this, it's simpler to do:
```rust
#[cfg(...)]
sockopt_impl!(...)
```